### PR TITLE
test

### PR DESCRIPTION
**Pull request by wen-hsiu-hsu**
testtttt

### CODE REVIEW

**Comment by wen-hsiu-hsu** [comment]
test

File: index.js [hunk 1]
```diff
@@ -1,21 +1,36 @@
-const core = require('@actions/core');
-const wait = require('./wait');
-
+const core = require("@actions/core");
+const github = require("@actions/github");
 
 // most @actions toolkit packages have async methods
 async function run() {
```
Comment (wen-hsiu-hsu) [inline index.js:4]: test review

**Review by wen-hsiu-hsu — COMMENTED** [review]
test 2 review

File: index.js (hunk 1 above)
Comment (hws-tw) [reply index.js:4]: test~

**Pull request closed without merging**